CONTRIBUTING.md

```diff
@@ -0,0 +1,148 @@
+# Contributing to GitLab Terraform Provider
+
+Thank you for contributing to this provider! :tada: :heart: :trophy: 
+
+Generally we accept any change that adds or changes a Terraform resource that is in line with the [GitLab API](https://docs.gitlab.com/ee/api/api_resources.html). It is always best to [open an issue](https://github.com/gitlabhq/terraform-provider-gitlab/issues/new/choose) before starting on a change.
+
+## Getting Started
+
+Use HashiCorp's [Plugin Development](https://www.terraform.io/plugin) guide as a reference, especially the [Provider Design Principles](https://www.terraform.io/plugin/hashicorp-provider-design-principles).
+
+See the [Developing The Provider](#developing-the-provider) section below for specifics about this GitLab provider.
+
+## PR Checklist
```

Comment:
Maybe we should also add this into a PR template.

Reply:
Opened https://github.com/gitlabhq/terraform-provider-gitlab/pull/788